src/cc/lib/graph/graph.cc

```diff
@@ -99,8 +99,14 @@ void Graph::GetNodeType(std::span<const NodeId> node_ids, std::span<Type> output
         }
         else
         {
-            const auto index = internal_id->second;
-            *curr_type = m_partitions[m_partitions_indices[index]].GetNodeType(m_internal_indices[index]);
+            auto index = internal_id->second;
+            size_t partition_count = m_counts[index];
+            for (size_t partition = 0; partition < partition_count; ++partition, ++index)
+            {
+                *curr_type = m_partitions[m_partitions_indices[index]].GetNodeType(m_internal_indices[index]);
+                if (*curr_type != snark::DEFAULT_NODE_TYPE)
```

Comment:
Why there are 2 default types here? One is passed as an argument, and another is a constant. Rename to `[missing|stub|...]_node_type`?

Reply:
Change to placeholder_node_type since nodes with this type are placeholder for edge sorting.